README.rst

```diff
@@ -39,13 +39,15 @@ To run the tests::
 Translations
 ------------
 
-To update translations from Transifex, `pip install transifex-client` and run::
+To update translations from Transifex, run::
 
     make txpull
     python manage.py compilemessages
 
 To push new strings to Transifex, run::
 
+    python manage.py makemessages -l en
+    # set "nplurals" and "plural" in djangopeople/locale/en/LC_MESSAGES/django.po
```

Comment:
If you could explain why this step is necessary, I might learn something. Otherwise, I suggest to squash this fixup commit into the first commit and merge the branch.